### PR TITLE
Update set-up-mdt-for-bitlocker.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Added spacing for Notes markdown.
Changed code block syntax to dos.
Removed bullet point regarding 'Turn on TPM Backup to A...', as it is deprecated.
Removed inline anchors.

### CODE REVIEW

File: windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker.md [hunk 1]
```diff
@@ -19,29 +19,34 @@ ms.topic: article
...
 
->[!NOTE]
->Depending on the Active Directory Schema version, you might need to update the Schema before you can store BitLocker information in Active Directory.
- 
-In Windows Server 2012 R2 (as well as in Windows Server 2008 R2 and Windows Server 2012), you have access to the BitLocker Drive Encryption Administration Utilities features, which will help you manage BitLocker. When you install the features, the BitLocker Active Directory Recovery Password Viewer is included, and it extends Active Directory Users and Computers with BitLocker Recovery information.
+> [!NOTE]
+> Depending on the Active Directory Schema version, you might need to update the Schema before you can store BitLocker information in Active Directory.
+
+In Windows Server version from 2008 R2 and onwards, you have access to the BitLocker Drive Encryption Administration Utilities features, which will help you manage BitLocker. When you install the features, the BitLocker Active Directory Recovery Password Viewer is included, and it extends Active Directory Users and Computers with BitLocker Recovery information.
```
Comment (illfated) [inline windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker.md]: I suggest using either "Windows Server version 2008 R2 and onward" or "Windows Server from version 2008 R2 and onward":
```suggestion
In Windows Server version 2008 R2 and onward, you have access to the BitLocker Drive Encryption Administration Utilities features, which will help you manage BitLocker. When you install the features, the BitLocker Active Directory Recovery Password Viewer is included, and it extends Active Directory Users and Computers with BitLocker Recovery information.
```
(Please note: "onwards" is the adverb (or used functionally as a verb in the imperative form), while _onward_ is the adjective.

**Comment by greg-lindsay** [comment]
I don't know who originally wrote this, but 'onwards' is not the appropriate term here. It should say "later." Onwards sounds like the operating system is a journey somewhere.

File: windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker.md [hunk 2]
```diff
@@ -19,29 +19,34 @@ ms.topic: article
...
 
->[!NOTE]
->Depending on the Active Directory Schema version, you might need to update the Schema before you can store BitLocker information in Active Directory.
- 
-In Windows Server 2012 R2 (as well as in Windows Server 2008 R2 and Windows Server 2012), you have access to the BitLocker Drive Encryption Administration Utilities features, which will help you manage BitLocker. When you install the features, the BitLocker Active Directory Recovery Password Viewer is included, and it extends Active Directory Users and Computers with BitLocker Recovery information.
+> [!NOTE]
+> Depending on the Active Directory Schema version, you might need to update the Schema before you can store BitLocker information in Active Directory.
+
+In Windows Server version from 2008 R2 and later, you have access to the BitLocker Drive Encryption Administration Utilities features, which will help you manage BitLocker. When you install the features, the BitLocker Active Directory Recovery Password Viewer is included, and it extends Active Directory Users and Computers with BitLocker Recovery information.
```
Comment (illfated) [inline windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker.md]: Either remove "from" or reverse the order (Windows Server from version 2008 R2 and later)
```suggestion
In Windows Server version 2008 R2 and later, you have access to the BitLocker Drive Encryption Administration Utilities features, which will help you manage BitLocker. When you install the features, the BitLocker Active Directory Recovery Password Viewer is included, and it extends Active Directory Users and Computers with BitLocker Recovery information.
```

Comment (illfated) [reply windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker.md]: D'oh. I will have to make a new PR for my belated suggestion.

**Comment by baardhermansen** [comment]
Sorry, I'm not used to pull requests being approved this fast :P  @illfated, yeah, I agree with your comment. It's what happens when someone who's first language isn't English...

**Pull request merged**